modules/nw-cudn-localnet.adoc

```diff
@@ -0,0 +1,79 @@
+//module included in the following assembly:
+//
+// * networking/multiple_networks/primary_networks/about-user-defined-networks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-cudn-localnet_{context}"]
+= Creating a ClusterUserDefinedNetwork CR for Localnet
+
+A `Localnet` topology connects the secondary network to the physical underlay. This enables both east-west cluster traffic and access to services running outside the cluster, but it requires additional configuration of the underlying Open vSwitch (OVS) system on cluster nodes.
```

Comment:
nit: 
> but it requires additional configuration of the underlying Open vSwitch (OVS) system on cluster nodes.

I would ommit "but", it may sounds like a down side for using this feature.

Can we have a a link for configuring the OVN bridge mapping? (i.e. the underlaying OVS on cluster nodes)

Reply:
Agreed. I'd write it more like "note that it requires additional configuration"